client/src/components/CreateJob/createjob.js

```diff
@@ -1,14 +1,18 @@
 import React, { Component } from "react";
 import Jumbotron from "../Jumbotron";
+import { List } from "../List";
 import API from "../../utils/API";
-import { Col, Row, Container } from "../Grid";
-
+import { Container } from "../Grid";
 import { Input, TextArea, FormBtn } from "../Form";
 import { Redirect } from "react-router-dom";
-import ListJobs from "../ListJobs";
+import Job from "../Job";
+import Button from "react-bootstrap/Button";
+import { Col, Row, Modal } from 'react-materialize';
+import "./style.css";
 
 class CreateJob extends Component {
```

Comment:
again naming convention is poor. CreateJob modal/form is better

Reply:
I saw that you guys have CreateJob and createJobForm or something like that. im confused on what this component actually is. i get what it does, it creates a job but is this a whole page??

Reply:
I think the important distinction here is that a component is a thing, so it should be named a noun (with descriptive modifiers) rather than a verb, like 'addChild'. A function makes sense as a verb if it performs an action, but components should be conceived of as nouns, not verbs. A similar comparison can be made with RESTful resources as nouns, and HTTP methods (get, post, put, delete, update) as verbs. So in the same way an endpoint url ideally wouldn't be "addChild," because the action would be described by the method, and the endpoint would just reference the noun, "child."

Reply:
Thanks for the input Darren and Alex. We will make note of it for future projects. 
-- we will clean up the duplicate createjob/ createjobform